### DISCUSSION TITLE
App Won't Open on Mac OS

### DISCUSSION THREAD

**Discussion by Bo390** [DeFiCh/app#876] [Q&A] [answered]
Mac OS 11.3.1. No version of the app from 2.3.2-2.3.5 will open. E.g. delete app and delete all folders from App Support. Install app, open app and accept security warning. I can see the app menu at the top left of the screen but no window opens on any of those versions. I have not had access to my coins for going on two weeks now. Can you please advise? Thanks.

**Comment by izzycsy** [comment, marked as answer]
Hi @Bo390 , is [this issue](https://github.com/DeFiCh/app/issues/765) the same as yours? It has been fixed in v2.3.5. Please keep your wallet.dat safe before proceeding to delete any hidden files.

**Reply by Bo390** [reply]
Yes, that fixed it. Thanks for the help.
